services/supervisord/supervisord.go

```diff
@@ -486,6 +487,23 @@ func (s *Service) UpdateConfiguration(settings *models.Settings) error {
 // once we start generating it. See alertmanager service.
 
 var templates = template.Must(template.New("").Option("missingkey=error").Parse(`
+{{define "dbaas-controller"}}
+[program:dbaas-controller]
+priority = 7
```

Comment:
Any particular reason we use the same priority as Prometheus?

Reply:
Please tell me what priority I should set. I dont know how much important it should be. Thanks.

Reply:
It defines the order of services startup. If priority is the same, services will be started at the same time. If we don't have a reason to do that, let's use a separate priority. Prometheus and Alertmanager use 7 and 8 below, so – 6?

Reply:
@AlekSi I changed it to 6 because here is space between dash, but if you really thought minus 6 then I will change it again. Thanks.

Reply:
No, that's dash. I probably used Russian punctuation rules :)